libmscore/clef.cpp

```diff
@@ -154,7 +154,7 @@ void Clef::layout()
       qreal yoff     = 0.0;
       if (clefType() !=  ClefType::INVALID && clefType() !=  ClefType::MAX) {
             symId = ClefInfo::symId(clefType());
-            yoff = lineDist * (lines - ClefInfo::line(clefType()));
+            yoff = lineDist * (5 - ClefInfo::line(clefType())); 
```

Comment:
No need to adapt clef to number of staff lines.  Since <ClefInfo::line> is counted from bottom, and staff lines are also removed from bottom, this causes the clef to move to a wrong pitch line.